repos/system_upgrade/el7toel8/actors/checkfstab/actor.py

```diff
@@ -0,0 +1,81 @@
+import os
+import time
+
+from leapp.actors import Actor
+from leapp.exceptions import StopActorExecutionError
+from leapp.libraries.common.fstab import drop_xfs_options
+from leapp.libraries.stdlib import api
+from leapp.models import Report, FstabSignal
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+from leapp import reporting
+
+
+PREFACE = """# Below are the modified contents of /etc/fstab that have been generated for
+# the upgraded system. Changes to this file will not be reflected. If you
+# believe the file has been modified incorrectly, please report a bug and
+# attach this file and the current /etc/ftsab.
```

Comment:
The information that user should check the file and in case of invalid content the bu should be reported is only here. This should be already part of the report as well - how a user realize they are supposed to check the file?

As well, in case there is a bug in the generated / modified fstab, we should encourage users to update the fstab so the removed options are not present and run leapp again - so we will not do the modifications.